admin_guide/backup_restore.adoc

```diff
@@ -133,8 +139,24 @@ inside the container.
 . Copy the *_db_* file over to the backup you created:
 +
 ----
-$ cp "$ETCD_DATA_DIR"/member/snap/db "$HOTDIR"/member/snap/db'
+# cp "$ETCD_DATA_DIR"/member/snap/db "$HOTDIR"/member/snap/db'
+----
+
+[[registry-certificates-backup]]
+=== Registry Certificates Backup
+
+. Save all the registry certificates, on every master and node host.
++
 ----
+# cd /etc/docker/certs.d/
+# tar cf /tmp/docker-registry-certs-$(hostname).tar * 
+----
++
+[NOTE]
+====
+When working with one or more xref:../install_config/registry/securing_and_exposing_registry#exposing-the-registry[external secured registry], 
```

Comment:
Hi @ahardin-rh, Thanks for your correction. 

But in this case, this only makes sense when the external registry is secured (it serves traffic via TLS) https://docs.openshift.com/container-platform/3.3/install_config/registry/securing_and_exposing_registry.html#securing-the-registry.
 
That's why certs need to be backed up. So I would keep the [external secured registry] instead of [external registry].